effects/rpc_known_smoldot_clients.ts

```diff
@@ -0,0 +1,32 @@
+import * as Z from "../deps/zones.ts"
+import * as rpc from "../rpc/mod.ts"
+import { rpcClient } from "./rpc.ts"
+
+function smoldotClient(relayChainSpecUrl: string, parachainSpecUrl?: string) {
+  return rpcClient(
+    rpc.smoldotProvider,
+    Z.lift([]).next(async () => {
```

Comment:
@harrysolovay used `Z.lift().next()` because `Z.call(async () => ...)` doesn't resolve to the correct type.

Should I create a PR to update `Z.call<R>` to `Z.call<Awaited<R>`